src/main/java/com/code_intelligence/jazzer/mutation/mutator/lang/ByteArrayMutatorFactory.java

```diff
@@ -89,7 +104,17 @@ public byte[] init(PseudoRandom prng) {
     public byte[] mutate(byte[] value, PseudoRandom prng) {
       // TODO: The way maxSizeIncrease is determined is just a heuristic and hasn't been
       //  benchmarked.
-      return LibFuzzerMutator.mutateDefault(value, Math.max(8, value.length / 16));
+      byte[] mutated = LibFuzzerMutator.mutateDefault(value, Math.max(8, value.length / 16));
```

Comment:
What I missed before: The second argument is the allowed length increase. We should probably cap it at `maxLength - value.length` to avoid unnecessary truncations.